IFC4x3/Templates/Object Composition/Aggregation/Spatial Composition/Documentation.md

```diff
@@ -1,9 +1,9 @@
 
-> NOTE&nbsp; The link between the highest level spatial element and the project is provided by this concept through _IfcRelContainedInSpatialStructure_ and not through declaration using _IfcRelDeclares_. This is a known anomaly intruduced to maintain compatibility with earlier versions of this standard.
+> NOTE&nbsp; The link between the highest level spatial element(s) and the project is provided by this concept through _IfcRelAggregates_ and not through declaration using _IfcRelDeclares_. This is a known anomaly intruduced to maintain compatibility with earlier versions of this standard.
 
-The order of spatial structure elements being included in the concept for builing projects are from high to low level: _IfcProject_, _IfcSite_, _IfcBuilding_, _IfcBuildingStorey_, and _IfcSpace_ with _IfcSite_, _IfcBuildingStorey_ and _IfcSpace_ being optional levels. Therefore an spatial structure element can only be part of an element at the same or higher level.
+The project spatial structure may be made up of a selection of different spatial structure elements with the most generic and simplest form from high to low level as follows: _IfcProject_, _IfcSite_, _IfcFacility_ (or any of its subtypes), _IfcFacilityPart_ (or _IfcBuildingStorey in the case of buildings), and _IfcSpace_ with _IfcSite_, _FacilityPart_ and _IfcSpace_ being optional levels. Therefore a spatial structure element should only be part of an element at the same or higher level, with the exception of _IfcFacility_ which can be part of an _IfcFacilityPart_ to allow for the regional or longitudinal division of a higjh level facility in to sections which can then contain one or more smaller functional facilities.
 
-In addition a more general hierarchical tree of spatial elements can be created by using _IfcSpatialZone_, from high to low: _IfcProject_, _IfcSite_, and _IfcSpatialZone_ with _IfcSite_ being an optional level.
+Where possible the relevent subtype of _IfcFacility_ should be used to describe the spatial structure element in question. When a adequate subtype of _IfcFacility_ with preefined or user defined type is not available the higher level generic _IfcFacility_ entity can be instantiated with the relevant and agreed typing identifier defined in _IfcFacility.ObjectType_. This allows for nearly full coverage of built environment domains and/or scenarios that have yet to be addressed with specific extensions.
 
-> NOTE&nbsp; The more general hiearchical tree has been introduced as an intermediate solution and stub for further extensions to support infrastructure works.
+In addition to the identified spatial structure elements _IfcSpatialZone_ can be used to provide cross domain or functional zones within a project, these elements can be mounted into the hierarchy in the same manner as other spatial structure elements, but should not aggregate any other spatial structure element types except other _IfcSpatialZone_ instances.
```

Comment:
Not agree, as far as I know the relation to mount SpatialZone into SpatialStructure is with IfcRelContainedInSpatialStructure and not IfcRelAggragate, that should be stated clearly ?

Reply:
You are correct @MatthieuPERIN, my mistake i miss read our diagram and have updated this.